test/features/fieldTypes.feature

```diff
@@ -69,6 +69,59 @@ Feature: Set of scenarios to check we can read and write to all field data types
       | dataType  |
       | Complex   |
 
```

Comment:
too much copy and pasting perhaps - some of these scenarios are for mid event callback i thought however all the titles say they are all for AboutToStart callback